config/initializers/data_storage_titles.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# whitelisted data storage titles definitions
+class DataStorageTitles
+  class << self
+    def list
+      @list ||= YAML.load_file(Barong::App.config.barong_config)['data_storage_titles']
```

Comment:
default it to [] if unset

Reply:
you mean if key `data_storage_titles` dont exist at all?

Reply:
yes to not brake on upgrade of barong if we don't update the configuration